frontend/src/apis/experiment/api.ts

```diff
@@ -191,45 +357,6 @@ export interface ProtobufEmpty {
  */
 export const ExperimentServiceApiFetchParamCreator = function (configuration?: Configuration) {
     return {
-        /**
-         * 
-         * @param {ApiExperiment} body 
-         * @param {*} [options] Override http request option.
-         * @throws {RequiredError}
-         */
-        createExperiment(body: ApiExperiment, options: any = {}): FetchArgs {
```

Comment:
Why are these removed?

Reply:
Thanks for the heads up. This was indeed messed up, because both create and list used the same endpoints with differing verbs (POST and GET).

I decided to go another route: encode the filter as a simple JSON string and put it in a single parameter. Maybe this was what you were alluding to earlier, and I think this should work better, as it doesn't break any of the existing semantics. See what you think.